src/commands/update_stacks_branch.yml

```diff
@@ -31,6 +34,14 @@ steps:
         [ -z "<<parameters.circle_tag>>" ] && export image=$CIRCLE_TAG
         [ -n "<<parameters.circle_tag>>" ] && export image=<<parameters.circle_tag>>
         echo "service is $service and image is $image"
-        ~/yq w -i docker-compose.<<parameters.stack>>.yml services.$service.image "heathmont/$service:$image"
+        if [ -z "<<parameters.stacks_services>>" ]
+        then
+          ~/yq w -i docker-compose.<<parameters.stack>>.yml services.$service.image "heathmont/$service:$image"
+        else
+          for i in <<parameters.stacks_services>>; do
+            echo "replacing image version for service $i"
+            ~/yq w -i docker-compose.<<parameters.stack>>.yml services.$i.image "heathmont/$service:$image"
```

Comment:
Shouldn't it be `heathmont/$i:$image` here?

Reply:
No, $service contains the correct repo name. we just need to use this image for different stacks services.

I now renamed "service" to "repo" for correctness.

Fragments from Bitcasino stack to illustrate this:
  frontend-apollo:
    image: heathmont/bitcasino-v3:v1.142.3
  sportsbet-casino:
    image: heathmont/bitcasino-v3:v1.142.3
  slotsio:
    image: heathmont/bitcasino-v3:v1.147.3
  luckyslots:
    image: heathmont/bitcasino-v3:v1.147.3